<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iHouse - pokój 2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pokój 2:</string>
    </property>
   </widget>
   <widget class="Line" name="pokoj1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wentylacja: </string>
    </property>
   </widget>
   <widget class="QLabel" name="obr_drzwi1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>36</width>
      <height>58</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="wywolaj_ruch1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj ruch</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="swiatlo1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="obr_zarowki1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="wentylacja1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="otworz_okno1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Otwórz okno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="otworz_drzwi1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Otwórz drzwi</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ustaw_stezenie1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temperatura1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Światło:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygeneruj stężenie dymu: </string>
    </property>
   </widget>
   <widget class="QLabel" name="obr_okna1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>32</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="slave2.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>actionconnect</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="slave2.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>configure</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="slave2.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="slave2.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="slave2.qrc"/>
 </resources>
 <connections/>
</ui>
